### ISSUE TITLE
inconsistent a11y instrumentation of rectangular vs round buttons

### ISSUE THREAD

**Issue by pixelzoom** [phetsims/sun#459]
RectangularButtonView sets tagName, and it's therefore unnecessary to specify `tagName` in any of its subclasses:

```js
86 tagName: 'button',
```

RoundButtonView does not set `tagName`, and `tagName: 'button'` is duplicated in subclasses.

The same approach should be used for rectangular and round buttons, and the former approach seems superior.

**Comment by jessegreenberg** [comment]
Thanks for pointing out, I added `tagName: 'button'`RoundButtonView and removed `tagName` from subtypes. I tested sun demo and round buttons in sims with a11y, all are still working. Closing.
